app/lib/common/widgets/html_editor/html_editor.dart

```diff
@@ -211,6 +215,32 @@ class HtmlEditorState extends State<HtmlEditor> {
     return isDesktop ? desktopEditor(roomId) : mobileEditor(roomId);
   }
 
+  Map<String, BlockComponentBuilder> _buildBlockComponentBuilders() {
+    final map = {...standardBlockComponentBuilderMap};
+    map[ParagraphBlockKeys.type] = ParagraphBlockComponentBuilder(
+      // only show placeholder at first line
+      showPlaceholder: (editorState, node) {
```

Comment:
This is actual placeholder showing logic, but modifying it to state to make it disappear upon entering newline (without any content on first line) seems to be not allowed and throws exception.